### ISSUE TITLE
Get started in no time > 'ivre ipdata --download' wrong URL 

### ISSUE THREAD

**Issue by Underknowledge** [cea-sec/ivre#1144]
Please try to follow this template. Also, please format your issue
properly (particularly to report code or command line output). You may
want to read
[Github's basic writing and formatting syntax](https://help.github.com/articles/basic-writing-and-formatting-syntax/).

### IVRE Version

Insert here the output of the command `ivre version`, displayed as
code. For example:

```
$ iroot@6d51c3e6ee92:/ivre-share# ivre version
IVRE - Network recon framework
Copyright 2011 - 2020 Pierre LALET <pierre@droids-corp.org>
Version 0.9.16.dev163

Python 3.7.3 (default, Jan 22 2021, 20:04:44) 
[GCC 8.3.0]

Linux 6d51c3e6ee92 5.11.17-200.fc33.x86_64 #1 SMP Wed Apr 28 17:34:39 UTC 2021 x86_64

Dependencies:
    Python module pymongo: 3.7.1
    Python module sqlalchemy: missing
    Python module psycopg2: missing
    Python module cryptography: missing
    Python module krbV: missing
    Python module PIL: 5.4.1
    Python module MySQLdb: missing
    Python module dbus: missing
    Python module matplotlib: missing
    Python module bottle: 0.12.15
    Python module OpenSSL: missing
    Python module tinydb: missing

```

### Summary

Just trying to start and run the "Get started in no time" procedure with vagrant on a fedora VM. 
To be honest, I have a hard time. 
URL changed 

```log 
Downloading https://dev.maxmind.com/static/csv/codes/iso3166.csv to /usr/local/share/ivre/geoip/iso3166.csv: Traceback (most recent call last):
  File "/usr/local/bin/ivre", line 103, in <module>
    main()
  File "/usr/local/bin/ivre", line 71, in main
    tools.get_command(next(iter(possible_commands)))()
  File "/usr/local/lib/python3.7/dist-packages/ivre/tools/ipdata.py", line 51, in main
    geoiputils.download_all(verbose=not args.quiet)
  File "/usr/local/lib/python3.7/dist-packages/ivre/geoiputils.py", line 184, in download_all
    udesc = opener.open(url)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found
```
somewhere around here? 
https://github.com/cea-sec/ivre/blob/787659a019ad4b1c86f8ae4b0526abd894bc4e17/ivre/geoiputils.py#L176

https://dev.maxmind.com/static/csv/codes/iso3166.csv
is now 
https://dev.maxmind.com/csv-files/codes/iso3166.csv?lang=en

### Expected behavior

dl file, letme scan

### Actual behavior

above

### How to reproduce

```bash 
mkdir -m 1777 var_{lib,log}_mongodb ivre-share
chcon -Rt svirt_sandbox_file_t var_{lib,log}_mongodb ivre-share
vagrant up --no-parallel
# some errors bout no ssh support for the container 
  Bringing machine 'ivredb' up with 'docker' provider...
  Bringing machine 'ivreweb' up with 'docker' provider...
  Bringing machine 'ivreclient' up with 'docker' provider...
  ==> ivredb: Creating and configuring docker networks...
  ==> ivredb: Creating the container...
      ivredb:   Name: ivredb
      ivredb:  Image: ivre/db
      ivredb: Volume: /root/nmap-converter/var_lib_mongodb:/var/lib/mongodb
      ivredb: Volume: /root/nmap-converter/var_log_mongodb:/var/log/mongodb
      ivredb:  
      ivredb: Container created: 6d92b9d27e031799
  ==> ivredb: Enabling network interfaces...
  ==> ivredb: Starting container...
  ==> ivredb: Provisioners will not be run since container doesn't support SSH.
  ==> ivreweb: Creating and configuring docker networks...
  ==> ivreweb: Creating the container...
      ivreweb:   Name: ivreweb
      ivreweb:  Image: ivre/web
      ivreweb:   Port: 80:80
      ivreweb:   Link: ivredb:ivredb
      ivreweb:  
      ivreweb: Container created: fceddeb0d5ce11cc
  ==> ivreweb: Enabling network interfaces...
  ==> ivreweb: Starting container...
  ==> ivreweb: Provisioners will not be run since container doesn't support SSH.
  ==> ivreclient: Creating and configuring docker networks...
  ==> ivreclient: Creating the container...
      ivreclient:   Name: ivreclient
      ivreclient:  Image: ivre/client
      ivreclient: Volume: /root/nmap-converter/ivre-share:/ivre-share
      ivreclient:   Link: ivredb:ivredb
      ivreclient:  
      ivreclient: Container created: 96d2ae20c549857b
  ==> ivreclient: Enabling network interfaces...
  ==> ivreclient: Starting container...
  ==> ivreclient: Provisioners will not be run since container doesn't support SSH.
docker exec -it ivreclient bash
yes | ivre ipinfo --init
yes | ivre scancli --init
yes | ivre view --init
yes | ivre flowcli --init
yes | ivre runscansagentdb --init
ivre ipdata --download                                                                                                                                             
  Downloading https://ivre.rocks/data/geolite/GeoLite2-City.tar.gz to /usr/local/share/ivre/geoip/GeoLite2-City.tar.gz:                                                                     
  done.
  ...                                                                                                           
  Downloading https://ivre.rocks/data/geolite/GeoLite2-dumps.tar.gz to /usr/local/share/ivre/geoip/GeoLite2-dumps.tar.gz: 
  done.
  Downloading https://dev.maxmind.com/static/csv/codes/iso3166.csv to /usr/local/share/ivre/geoip/iso3166.csv: Traceback (most recent call last):
    File "/usr/local/bin/ivre", line 103, in <module>
      main()
    File "/usr/local/bin/ivre", line 71, in main
      tools.get_command(next(iter(possible_commands)))()
    File "/usr/local/lib/python3.7/dist-packages/ivre/tools/ipdata.py", line 51, in main
      geoiputils.download_all(verbose=not args.quiet)
    File "/usr/local/lib/python3.7/dist-packages/ivre/geoiputils.py", line 184, in download_all
      udesc = opener.open(url)
    File "/usr/lib/python3.7/urllib/request.py", line 531, in open
      response = meth(req, response)
    File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
      'http', request, response, code, msg, hdrs)
    File "/usr/lib/python3.7/urllib/request.py", line 569, in error
      return self._call_chain(*args)
    File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
      result = func(*args)
    File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
      raise HTTPError(req.full_url, code, msg, hdrs, fp)
  urllib.error.HTTPError: HTTP Error 404: Not Found
  
```

**Comment by p-l-** [comment]
Sorry you had "a hard time", but it seems that you are running an old version of the master branch. I **think** this has been fixed in #1128, a bit less than a month ago.

**Comment by Underknowledge** [comment]
Thanks a lot for pushing! Damn awesome
I'm able to get scans into the UI <3 

For curious ones after me who want to quickly scan their homenet
```bash
export NETWORK=$(routel 2> /dev/null | grep default -A 1 | awk 'FNR==2{print $1 }' | sed 's/\/*$//g' ) && \
export SUBNETMASK=$(routel  2> /dev/null | grep default -A 1 | awk 'FNR==2{print $2 }' | sed 's/\/*$//g' ) && \
export HOSTN=$(hostname) && \
export RUN="$(date +"%Y%m%d-%H%M")" && \
docker exec -it ivreclient bash -c "echo \"Starting scan with prefix ${RUN}\" && \
mkdir -p /tmp/${RUN} && \
cd /tmp/${RUN} && \
echo ${RUN} | tee -a testintmp && ls /tmp/${RUN} && pwd && \
ivre runscans --nmap-template default --network ${NETWORK}/${SUBNETMASK} && \
ivre scan2db -c NET,NET-CAMPAIGN-${RUN} -s ${HOSTN} -r scans/NET-${NETWORK}-${SUBNETMASK}/up/ && \
(ivre db2view --no-merge nmap --category NET,NET-CAMPAIGN-${RUN} || ivre db2view nmap)
"
```
